tests/units/typing/url/test_video_url.py

```diff
@@ -46,12 +47,15 @@ def test_load(file_url):
     'file_url',
     [LOCAL_VIDEO_FILE, REMOTE_VIDEO_FILE],
 )
-def test_load_key_frames(file_url):
+@pytest.mark.parametrize(
+    'field',
+    [f for f in VideoLoadResult._fields],
+)
+def test_load_one_of_named_tuple_results(file_url, field):
     url = parse_obj_as(VideoUrl, file_url)
-    key_frames = url.load_key_frames()
+    result = getattr(url.load(), field)
```

Comment:
ah yes I did not saw it

Reply:
Lets not use VideoLoadResult._fields here. Lets make the list ourself. Otherwise if by mistake we change smth on the _fields side the test will catch it.